WordPress/Classes/ViewRelated/Post/PostEditor.swift

```diff
@@ -111,6 +111,9 @@ protocol PostEditor: class, UIViewControllerTransitioningDelegate {
     var autosaver: Autosaver { get set }
     /// true if the post is the result of a reblog
     var postIsReblogged: Bool { get set }
+
+    /// Returns the word counts of the content in the editor.
+    var wordCount: UInt? { get }
```

Comment:
Would it make sense to have this non-optional and default to zero in Gutenberg-side implementation?

In this case I'm not sure what would be the difference between `nil` and 0, and seems like needed just because of the special case of how we get the count on Gutenberg.